dgx-2-test.job

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#SBATCH -p dgx2q                   # partition (GPU queue)
+#SBATCH -N 1                       # number of nodes
+#SBATCH -n 4                       # number of cores
+#SBATCH -w g001                    # DGX-2 node
+#SBATCH --gres=gpu:4               # number of V100's
+#SBATCH --mem 10G                  # memory pool for all cores
+#SBATCH -t 0-00:10                 # time (D-HH:MM)
+#SBATCH -o slurm.%N.%j.out  # STDOUT
+#SBATCH -e slurm.%N.%j.err  # STDERR
+
+ulimit -s 10240
+module load slurm 
+module load openmpi/4.0.1
+module load cuda10.1/toolkit/10.1.243
+
+# Check how many gpu's your job got
+#nvidia-smi
+
+## Copy input files to the work directory:
+mkdir -p /work/$USER/ShallowWaterGPU
```

Comment:
Jeg liker å ha et script dir som følger, og bruke det istedenfor $USER og $HOME. Da er det mulig for meg å installere noe og deg å kjøre det. (sånn jeg har satt opp på f.eks. Nebula, se https://github.com/metno/VolcanicAshInversion/blob/master/inversion_job_script.sh)

Men det kan vi ta senere. Mest FYI for deg for andre muligheter for å sette opp kjøringer.